src/lib/layout/layout.ts

```diff
@@ -410,3 +417,33 @@ function getBuildOutput(buildOutput: string | undefined, scanResult: ScanResult)
 
   return Path.join(scanResult.projectRoot, output)
 }
+
+export function findNexusModules(tsConfig: ScanResult['tsConfig'], maybeAppModule: string | null): string[] {
+  try {
+    log.trace('finding nexus modules')
+    const project = new ts.Project({
+      addFilesFromTsConfig: false, // Prevent ts-morph from re-parsing the tsconfig
+    })
+
+    tsConfig.content.fileNames.forEach((f) => project.addSourceFileAtPath(f))
+
+    const modules = project
+      .getSourceFiles()
+      .filter((s) => {
+        // Do not add app module to nexus modules
+        if (s.getFilePath().toString() === maybeAppModule) {
+          return false
+        }
+
+        return s.getImportDeclaration('nexus') !== undefined
+      })
+      .map((s) => s.getFilePath().toString())
+
+    log.trace('done finding nexus modules', { modules })
+
+    return modules
+  } catch (error) {
+    log.error('We could not find your nexus modules', { error })
```

Comment:
Not fatal because the user will just see the message, add some modules, try again, right?

Reply:
Exactly!